CONTRIBUTING.md

```diff
@@ -278,8 +278,22 @@ git push $REMOTE master --follow-tags
 # Publish to NPM.
 # Make sure you are in the directory of the package you want to publish.
 yarn publish
+
+# Congratulations! You should see successfully published packages.
+#
+# You may also need to do some post-publish administration and close
+# the current Milestone in the github interface,
```

Comment:
github → GitHub if you can be bothered

![asking-for-a-friend](https://user-images.githubusercontent.com/7074/115360316-cf14bb80-a1bf-11eb-8249-1beae9732c2b.gif)